application/libraries/REST_Controller.php

```diff
@@ -14,7 +14,7 @@
  * @link            https://github.com/chriskacerguis/codeigniter-restserver
  * @version         3.0.0
  */
-abstract class REST_Controller extends CI_Controller {
+abstract class REST_Controller extends MX_Controller {
```

Comment:
Is the HMVC, change to CI_Controller.
On 14 Apr 2016 10:30 pm, "Chris Kacerguis" <notifications@github.com> wrote:

> In application/libraries/REST_Controller.php
> <https://github.com/chriskacerguis/codeigniter-restserver/pull/642#discussion_r59785980>
> :
>
> > @@ -14,7 +14,7 @@
> >   * @link            https://github.com/chriskacerguis/codeigniter-restserver
> >   * @version         3.0.0
> >   */
> > -abstract class REST_Controller extends CI_Controller {
> > +abstract class REST_Controller extends MX_Controller {
>
> What is MX_Controller?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/chriskacerguis/codeigniter-restserver/pull/642/files/debb7c683ddb297120a385cf98a12ea9c4737baa#r59785980>
>